src/controllers/tile.controller.ts

```diff
@@ -70,7 +70,8 @@ export default class UserController {
     }
 
     console.log("Generating road network...");
-    const network = new JsStreetNetwork(osmXML, {
+    const boundaryGeojson = "";
```

Comment:
When this is blank, it'll use `<bounds>` from the OSM.XML, or compute a bounding box from all the elements if that's missing. As we've been discussing in the other issue, we may need to explicitly pass a rectangle covering the tile here. And play around with some buffer around the "actual" tile size